packages/protocol/contracts/governance/Election.sol

```diff
@@ -294,6 +294,44 @@ contract Election is
     return true;
   }
 
+  function _revokeVotes(
+    address account,
+    address group,
+    uint256 value,
+    address lesser,
+    address greater,
+    uint256 index
+  ) public returns (uint256) {
```

Comment:
Change to internal?

Reply:
Yup good call, this was a remnant from testing, fixed.